src/diet/index.ts

```diff
@@ -21,6 +20,10 @@ import { get as getModifier } from "../modifier";
 import { get } from "../property";
 import { $effect, $item, $items, $skill, $stat } from "../template-string";
 import { sum } from "../utils";
+import { Mayo, installed as mayoInstalled } from "../resources/2015/MayoClinic";
+
+export type DietItem<T> = [MenuItem<T>[], number];
```

Comment:
I think this should be a class, so we can separate the helpers and main item instead of having them all in one array. I think garbo is our only consumer so it shouldn't be a problem that it's a breaking change.